packages/paste-website/src/pages/components/chat-composer/index.mdx

```diff
@@ -62,122 +81,426 @@ export const getStaticProps = async () => {
+
+#### Variants
+
+ChatComposerContainer has `default` and `contained` variants. The contained variant is used as part of the new AI experience.
+
 ## Examples
 
-### With placeholder
+### ChatComposer with placeholder
 
 Set a placeholder value using a `placeholder` prop.
 
-<LivePreview scope={{ChatComposer}} language="jsx">
+<LivePreview scope={{ ChatComposer }} language="jsx">
   {`<ChatComposer config={{namespace: 'customer-chat', onError: (e) => { throw e } }} placeholder="Chat text" ariaLabel="A placeholder chat composer" />`}
 </LivePreview>
 
-### With initial value
+### ChatComposer with initial value
 
 Set an initial value using an `initialValue` prop. This prop is limited to providing single line strings. For more complicated initial values interact with the Lexical API directly
 using the `config` prop and `editorState` callback.
 
-<LivePreview scope={{ChatComposer}} language="jsx">
+<LivePreview scope={{ ChatComposer }} language="jsx">
   {`<ChatComposer config={{namespace: 'customer-chat', onError: (e) => { throw e } }} initialValue="This is my initial value" ariaLabel="An initial value chat composer" />`}
 </LivePreview>
 
-### With max height
+### ChatCompoer with max height
 
 Restrict the height of the composer using a `maxHeight` prop.
 
-<LivePreview noInline language="jsx" scope={{ChatComposer, $getRoot, $createParagraphNode, $createTextNode}}>
+<LivePreview noInline language="jsx" scope={{ ChatComposer, $getRoot, $createParagraphNode, $createTextNode }}>
   {MaxHeightExample}
 </LivePreview>
 
-### With rich text
+### ChatComposer with rich text
 
 Set a rich text value using one of the Lexical formatting APIs such as [`toggleFormat`](https://lexical.dev/docs/api/classes/lexical.TextNode#toggleformat)
 
-<LivePreview noInline language="jsx" scope={{ChatComposer, $getRoot, $createParagraphNode, $createTextNode}}>
+<LivePreview noInline language="jsx" scope={{ ChatComposer, $getRoot, $createParagraphNode, $createTextNode }}>
   {RichTextExample}
 </LivePreview>
 
-### With Chat Log
+### Chat Composer with responsive attachments
 
-Use Chat Composer alongside other Paste components such as [Chat Log](/components/chat-log) to build more complex chat UI.
+Chat composer components suite demonstrating responsive attachments
 
 <LivePreview
   noInline
   language="jsx"
   scope={{
-    Box,
-    ChatLogger,
-    ChatLog,
-    ChatBookend,
-    ChatBookendItem,
-    ChatEvent,
-    ChatMessage,
-    ChatBubble,
-    ChatMessageMeta,
-    Avatar,
-    ChatAttachment,
-    ChatAttachmentDescription,
-    ChatAttachmentLink,
-    ChatMessageMetaItem,
+    SendIcon,
+    Button,
+    SendIcon,
+    AttachIcon,
+    DownloadIcon,
     ChatComposer,
+    ChatComposerContainer,
+    ChatComposerActionGroup,
+    ChatComposerAttachmentGroup,
+    ChatComposerAttachmentCard,
+    ChatComposerAttachmentLink,
+    ChatComposerAttachmentDescription,
+  }}
+>
+  {ResponsiveContainedAttachmentsExample}
+</LivePreview>
+
```

Comment:
suggestion(blocking): Add a section showing the default contained variant OR update the current "disabled contained variant" section to explain and show both the default and disabled states.